spec/index.html

```diff
@@ -283,9 +283,158 @@ <h2>Introduction</h2>
   <section id="basic-concepts" class="informative">
     <h2>Basic Concepts</h2>
 
-    <p>FIXME.</p>
+    <p>
+      To ease writing and collaborating on JSON-LD documents, it is a common practice
+      to serialize them as YAML.
+      This requires a registered media type, not only to enable content negotiation
+      of linked data documents in YAML, but even to define the expected behavior of
+      applications that processes these documents, including fragment identifiers and
+      interoperability considerations.
+    </p>
+
+    <p>
+      This is because YAML is more flexible than JSON:
+    </p>
+
+      <ul>
+        <li>YAML supports different encodings, including UTF-8, UTF-16, and UTF-32.</li>
+        <li>YAML supports more data types than JSON.</li>
+        <li>the structure of a YAML document &mdash; that is, a named YAML representation graph &mdash;
+        is a tree that can have cycles.</li>
+
+    <p>
+      The first goal of this specification is to allow a JSON-LD document to be
+      processed and serialized into YAML, and then back into JSON-LD, without
+      losing any information.
+
+      This is always possible, because a YAML representation graph can always represent
+      a non-cyclic tree, because JSON data types are a subset of YAML's, and because
```

Comment:
```suggestion
        is a rooted, directed graph that can have cycles.</li>

    <p>
      The first goal of this specification is to allow a JSON-LD document to be
      processed and serialized into YAML, and then back into JSON-LD, without
      losing any information.

      This is always possible, because a YAML representation graph can always represent
      a tree, because JSON data types are a subset of YAML's, and because
```